### PR TITLE
Added connection with Front-End and Back-End

### PR DESCRIPTION
**Pull request by itc02**

### CODE REVIEW

File: src/utils/requests/todos.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  status: number;
+  statusText: string;
+  headers: any;
+  config: any;
+  request: XMLHttpRequest;
+}
+
+
+export const GetTodos = () => {
+  const [ todos, setTodos ] = useState<any>([]);
```
Comment (arpinehp) [inline src/utils/requests/todos.tsx:25]: Use `Data` instead of `any`

File: src/utils/requests/todos.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  const fetchData = async () => {
+    const res = await axios.get<Response>('http://localhost:3000/todos');
+    setTodos(res.data);
+  }
+
+  useEffect(() => {
+    fetchData();
+  }, [])
```
Comment (arpinehp) [inline src/utils/requests/todos.tsx:34]: `todos`? and at the end of the line `;` please

File: src/utils/requests/todos.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  config: any;
+  request: XMLHttpRequest;
+}
+
+
+export const GetTodos = () => {
+  const [ todos, setTodos ] = useState<any>([]);
+
+  const fetchData = async () => {
+    const res = await axios.get<Response>('http://localhost:3000/todos');
```
Comment (arpinehp) [inline src/utils/requests/todos.tsx:28]: maybe
```
axios.get<Response>('http://localhost:3000/todos')
.then(res => setTodos(res.data))
.catch(error => console.log(error));
```

File: src/components/mainTable/MainTable.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      <thead>
+        <tr>
+          <th>Title</th>
+          <th>State</th>
+          <th>Assigned to</th>
+          <th>Deadline</th>
+        </tr>
+      </thead>
+      <tbody>
+        {todos.map((todo: any, index: number) => {
```
Comment (arpinehp) [inline src/components/mainTable/MainTable.tsx:19]: Use `Data` interface form src/utils/requests/todos.tsx

File: src/components/app/App.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+table, th, td {
```
Comment (arpinehp) [inline src/components/app/App.css:1]: remove?

File: src/components/mainTable/MainTable.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+table, th, td {
```
Comment (arpinehp) [inline src/components/mainTable/MainTable.css:1]: it's ok for now. when you will start to use `styled-components` don't forget to use it here as well

File: package.json [hunk 7]
```diff
@@ -1,17 +1,19 @@
 {
-  "name": "a",
+  "name": "todo",
```
Comment (arpinehp) [inline package.json:2]: :rocket:

File: src/utils/requests/todos.tsx (hunk 2 above)
Comment (itc02) [reply src/utils/requests/todos.tsx:34]: If I write `todos` in the brackets, `fetchData` will be called infinitely. But, in other hand, this syntax means that `fetchData` will be called once when component will be rendered, and if, for example, I add todo, new one will not be appear in the table until refreshing the page.

File: src/utils/requests/todos.tsx (hunk 1 above)
Comment (itc02) [reply src/utils/requests/todos.tsx:25]: Or <Array<Data>>?

**Pull request merged**